lib/matplotlib/widgets.py

```diff
@@ -2795,6 +2795,11 @@ def __init__(self, ax, onselect, drawtype='box',
         self.drag_from_anywhere = drag_from_anywhere
         self.ignore_event_outside = ignore_event_outside
 
+        # State to allow the option of an interactive selector that can't be
+        # interactively drawn. This is used in PolygonSelector as an
+        # interactive bounding box to allow the polygon to be easily resized
+        self._allow_creation = True
```

Comment:
Can `self._selection_completed` used instead?

Reply:
I tried that initially and it didn't work - there's a couple of places in the logic that I changed below that needed this new property.